WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -137,6 +138,17 @@ private extension OrderDetailsViewController {
         tableView.accessibilityIdentifier = "order-details-table-view"
     }
 
+    func configureStackView() {
+        stackView.layer.borderWidth = Constants.borderWidth
+        stackView.layer.borderColor = UIColor.border.cgColor
```

Comment:
Should we also move this border setup under `if` below? Or we need the border if the view is full width?

Reply:
I was 50/50 of making it flagged or not, but you're right. Changed here: https://github.com/woocommerce/woocommerce-ios/pull/11714/commits/768190332829be604ef719e8653eb45d935e9273